posts/2023-10-17-23.0.0.10.adoc

```diff
@@ -0,0 +1,162 @@
+---
+layout: post
+title: "Support for Java 21 in Open Liberty 23.0.0.10"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/lauracowen
+author_github: https://github.com/lauracowen
+seo-title: Support for Java 21 in Open Liberty 23.0.0.10 - OpenLiberty.io
+seo-description: Java 21 support, the first long-term support release since Java 17 two years ago. Try it now.
+blog_description: Java 21 support, the first long-term support release since Java 17 two years ago. Try it now.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= Support for Java 21 in Open Liberty 23.0.0.10
+Laura Cowen <https://github.com/lauracowen>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Java 21 is finally here! Java 21 is the first long-term support (LTS) release since Java 17 was released two years ago. It offers some new functionality and changes that you’ll want to check out for yourself. In particular, there's the introduction of Virtual Threads.
```

Comment:
```suggestion
Java 21 is finally here! Java 21 is the first long-term support (LTS) release since Java 17 was released two years ago. It offers some new functionality and changes that you’ll want to check out for yourself. In particular, there's the introduction of virtual threads.
```